.github/workflows/testing-new-gateway.yml

```diff
@@ -62,32 +58,14 @@ jobs:
           yarn run setup:dev
           yarn build
 
-      - name: Run tools (network)
-        run: |
-          docker login -u ${{ secrets.NEVERMINED_DOCKER_USERNAME }} -p ${{ secrets.NEVERMINED_DOCKER_TOKEN}}
-          sudo chmod go+r /etc/hosts
-          sudo echo "127.0.0.1  nevermined-metadata" | sudo tee -a /etc/hosts
-          git clone https://github.com/nevermined-io/tools
-          cd tools
-
-          export KEEPER_OWNER_ROLE_ADDRESS="0xe2DD09d719Da89e5a3D0F2549c7E24566e947260"
-          rm -rf "${HOME}/.nevermined/nevermined-contracts/artifacts"
-
-          # Disabling the compute api for now
-          # Check https://github.com/nevermined-io/sdk-js/issues/33
-          
-          # TODO: Use the default Node when merged into master
-          export KEEPER_VERSION=develop
-          export SUBGRAPH_VERSION=develop
-          export NODE_VERSION=develop
-          ./start_nevermined.sh ${{ matrix.network }} --no-graph --no-node --opengsn 2>&1 | tee nevermined_tools.txt &
-          cd ..
-          ./scripts/wait-nevermined.sh
-
       - name: Run node
+        env:
+          WEB3_PROVIDER_URL: http://contracts.nevermined.localnet
+          ESTUARY_TOKEN: ESTaa43688b-4ccf-4dad-8a16-410e488706ffARY
```

Comment:
this should come from a github secret

Reply:
it is a test token from the local estuary development node

Reply:
was tired of waiting for estuary so I added a node to nvm-tools. that token is only valid there